messaging/src/main/java/io/lantern/messaging/metadata/Metadata.kt

```diff
@@ -62,7 +62,7 @@ class Metadata(val mimeType: String?, val thumbnail: ByteArray?, val thumbnailMi
                 }
             } catch (t: Throwable) {
                 logger.error("couldn't extract thumbnail: ${t.message}")
-                return Metadata(mimeType, null, null)
+                return Metadata(mimeType, null, null, null)
```

Comment:
This im not quite sure if we need to inject the duration inside the thumbnail 🤔 or we can leave it as it is.